app/assets/stylesheets/pages.scss

```diff
@@ -0,0 +1,51 @@
+// Place all the styles related to the Pages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+
+
+/* General */
+html, body {
+  height: 100%;
+  margin: 0;
+}
+
+body {
+  //background-image: url('http://s3.amazonaws.com/codecademy-content/courses/learn-rails/img/bg-static.jpg');
+  background-image: url('https://scontent-dub4-1.xx.fbcdn.net/v/t1.0-9/67945814_2151005818530695_8137754632576303104_n.jpg?_nc_cat=108&_nc_ohc=M_A70dGsqAsAX8p1GmI&_nc_ht=scontent-dub4-1.xx&oh=cbe6bb4bcd2482543c7c8985c357f61c&oe=5EFED1AE');
```

Comment:
As you've downloaded this image as an asset, instead of referencing the web url you could reference your asset with `url('catz/catz.png')`

Reply:
I was struggling with the path, thanks for pointing me in the right direction.